spec/tags/core/method/parameters_tags.txt

```diff
@@ -1,2 +1 @@
 fails:Method#parameters returns [[:rest]] for a Method generated by respond_to_missing?
```

Comment:
This one should probably be easy to pass too, but I didn't look into it.

Reply:
We seem to use the shared method info for `:respond_to_missing?`. I didn't look at it more than that.

Reply:
It's actually not about `:respond_to_missing?`, the description is misleading.
It's for a Method generation by `foo.method(name)` when the method name does not exist but `foo.respond_to_missing?(name)` returns true.
In other words, it's also a generated SharedMethodInfo and should be trivial to fix.

Reply:
Okay. I'll add it to my TODO list. This PR fixes an immediate problem for me so I'd like to get it in.